paasta_tools/cli/utils.py

```diff
@@ -1013,6 +1014,20 @@ def get_instance_configs_for_service(service, soa_dir, type_filter=None):
                     soa_dir=soa_dir,
                     load_deployments=False,
                 )
+        if 'kubernetes' in type_filter:
```

Comment:
feels like this whole bit of code could be DRY'd up a little. but nbd, we can come back to it